commons/commands.py

```diff
@@ -513,6 +513,7 @@
                                 "--sort-by=.metadata.creationTimestamp -o " \
                                 "jsonpath='{{.items[-1:].metadata.name}}'"
 KUBECTL_GET_RPM = "kubectl exec -it {} -c {} -- rpm -qa|grep -i {}"
+GET_POD = "kubectl describe po {} | grep image"
```

Comment:
> [root@ssc-vm-g2-rhev4-2344 ~]# kubectl describe pods cortx-server-ssc-vm-g2-rhev4-2328-fc4ffbf7-4d4m5 | grep image
[root@ssc-vm-g2-rhev4-2344 ~]#
[root@ssc-vm-g2-rhev4-2344 ~]# kubectl describe pods  | grep image
[root@ssc-vm-g2-rhev4-2344 ~]#

Above command does not give any image info. 
Please add the correct command as well as the appropriate name.

Reply:
we will get the o/p if we have freshly deployed setup 
![image](https://user-images.githubusercontent.com/86817153/160614254-fdcc7ebd-d0c0-499b-973e-f24441c33219.png)
but if setup is old the Events section get overwrite. and latest event will populate not the one which are occured at deployment time.